testing/features/components/navigation.feature

```diff
@@ -29,12 +27,6 @@ Feature: Navigation component
       Then the Close button is present
       And the dropdown menu is open
 
-    @failing @NP-1755
```

Comment:
This does fail and this is one of the few components that need behavioural tests so I'd be reluctant to delete this

Reply:
The `@failing` tag means it's never run so will see if I can't wrangle things to get this running again.